beps/bep_0003.rst

```diff
@@ -87,70 +92,153 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layers``
```

Comment:
The client can get disjoint sets of peers for each announce though, so it may receive a peer only under the legacy infohash when it does in fact support both formats. The client would only discover the peer's support for the new format during the handshake, so we need a flag to signal that.